src/index.js

```diff
@@ -5,32 +5,48 @@ import 'file?name=index.html!../example/index.html';
 import 'react-toolbox/lib/commons.scss';
 import Root from './root';
 import registry from './lib/registry';
+import { forceTrailingSlash } from './lib/pathHelper';
 import './index.css';
 
 if (process.env.NODE_ENV !== 'production') {
   require('./utils.css'); // eslint-disable-line
 }
 
-// Create mount element dynamically
-const el = document.createElement('div');
-el.id = 'root';
-document.body.appendChild(el);
+const fixUrl = (pathname) => {
```

Comment:
I gave this a go (see new commit) -- but unfortunately it didn't work - if you have a URL `http://localhost/admin`, `http://localhost/admin/` or `http://localhost/admin//` the path is `/` as far React router is concerned.